content/webapp/views/components/InPageNavigation/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from 'react';
+
+import InPageNavigationSimple, {
+  Props as InPageNavigationSimpleProps,
+} from './InPageNavigation.Simple';
+import InPageNavigationSticky, {
+  Props as InPageNavigationStickyProps,
+} from './InPageNavigation.Sticky';
+
+type Props =
+  | (InPageNavigationSimpleProps & { variant: 'simple' })
+  | (InPageNavigationStickyProps & { variant: 'sticky' });
+
+const InPageNavigation: FunctionComponent<Props> = props => {
+  const { variant } = props;
+
+  if (variant === 'simple') {
+    return <InPageNavigationSimple {...props} />;
+  }
+
+  if (variant === 'sticky') {
+    return <InPageNavigationSticky {...props} />;
+  }
+
+  throw new Error(`Unknown variant: ${variant}`);
```

Comment:
Oooh! Is this dangerous to have on live? It's a nice warning if you're on the page where the variant has been removed but is there otherwise a danger of this going live and the whole page breaking for users?
Typescript should warn us enough?